cmd/detectExecuteScan.go

```diff
@@ -253,9 +254,24 @@ func exitCodeMapping(exitCodeKey int) string {
 
 func getDetectScript(config detectExecuteScanOptions, utils detectUtils) error {
 	if config.ScanOnChanges {
+		log.Entry().Infof("Using Detect Rescan script")
 		return utils.DownloadFile("https://raw.githubusercontent.com/blackducksoftware/detect_rescan/master/detect_rescan.sh", "detect.sh", nil, nil)
 	}
-	return utils.DownloadFile("https://detect.synopsys.com/detect.sh", "detect.sh", nil, nil)
+	if piperutils.ContainsStringPart(config.CustomEnvironmentVariables, "DETECT_LATEST_RELEASE_VERSION") {
+		releaseVersion := ""
+		for _, i := range config.CustomEnvironmentVariables {
+			if strings.Contains(i, "DETECT_LATEST_RELEASE_VERSION") {
+				releaseVersion = strings.Split(i, "=")[1]
+			}
+		}
+		log.Entry().Infof("Using detect script Version %v ", releaseVersion)
```

Comment:
This line will always return an empty string if the env variable is not set.
I would suggest moving it inside the If statement after line 264?